From 7d9bb3b5a1cf40782c3079a1d062a930655dc137 Mon Sep 17 00:00:00 2001
From: Leonidas Spyropoulos <artafinde@archlinux.org>
Date: Fri, 21 Oct 2022 11:15:19 +0100
Subject: [PATCH] Makefile: Remove version from /usr/share/makedumpfile

Version specific paths doesn't make sense at
/usr/share/makedumpfile. This assumes you will have only one version
installed which on a normal system it makes sense and devs can always
specify different DESTDIR per versions.

Fixes: #10

Signed-off-by: Leonidas Spyropoulos <artafinde@archlinux.org>
---
 Makefile | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/Makefile b/Makefile
index 548e5b7..f6ecbe2 100644
--- a/Makefile
+++ b/Makefile
@@ -130,6 +130,6 @@ install:
 	install -m 755 -t ${DESTDIR}/usr/sbin makedumpfile $(VPATH)makedumpfile-R.pl
 	install -m 644 -t ${DESTDIR}/usr/share/man/man8 makedumpfile.8
 	install -m 644 -t ${DESTDIR}/usr/share/man/man5 makedumpfile.conf.5
-	mkdir -p ${DESTDIR}/usr/share/makedumpfile-${VERSION}/eppic_scripts
-	install -m 644 -D $(VPATH)makedumpfile.conf ${DESTDIR}/usr/share/makedumpfile-${VERSION}/makedumpfile.conf.sample
-	install -m 644 -t ${DESTDIR}/usr/share/makedumpfile-${VERSION}/eppic_scripts/ $(VPATH)eppic_scripts/*
+	mkdir -p ${DESTDIR}/usr/share/makedumpfile/eppic_scripts
+	install -m 644 -D $(VPATH)makedumpfile.conf ${DESTDIR}/usr/share/makedumpfile/makedumpfile.conf.sample
+	install -m 644 -t ${DESTDIR}/usr/share/makedumpfile/eppic_scripts/ $(VPATH)eppic_scripts/*
-- 
2.38.1

